qiskit/opflow/converters/circuit_sampler.py

```diff
@@ -80,7 +80,7 @@ def __init__(self,
         self._check_quantum_instance_and_modes_consistent()
 
         # Object state variables
-        self._last_op = None
+        self._last_op = None  # type: Optional[OperatorBase]
```

Comment:
This is just to note. I believe since the min level of Python supported now is 3.6 these typehints can go directly on the variable and no longer need to be added as comments (which we needed to do in the past because we supported 3.5). I believe @manoelmarques has been changing these away from comments when he has been changing the code